src/omega_edit/client.ts

```diff
@@ -45,19 +45,21 @@ async function getOmegaEditPort(port: number | undefined = undefined) {
     const defaultServerPort = vscode.workspace
       .getConfiguration('dataEditor')
       .get<number>('defaultServerPort', serverPort)
```

Comment:
Can we rename `defaultServerPort` to `serverPort`. Since it can be updated I don't think it should have `default` in the name as that makes me thing this value shouldn't be set.